### DISCUSSION TITLE
Are we planning a New Release?

### DISCUSSION THREAD

**Discussion by JoelPelaezP** [spec-first/connexion#1924] [General]
Hi!,

Is there a new release scheduled for the next weeks? Looking forward to try this feature => https://github.com/spec-first/connexion/pull/1883

Thanks for all your support and hard work!

**Comment by chrisinmtown** [comment]
I realize this is old, for what it's worth, I see that version 3.1.0 was released on 27 May 2024 and the "What's Changed" page includes this item:

> Inject current request in security handlers by @RobbeSneyders in https://github.com/spec-first/connexion/pull/1883

So maybe you might like to close this discussion?  HTH

**Comment by JoelPelaezP** [comment]
Thanks for the update.
